google/api/http.proto

```diff
@@ -346,6 +347,20 @@ message HttpRule {
   // message type.
   string body = 7;
 
+  // A mapping from HTTP headers to request fields. If
+  // a header has a case-insensitive match to a key in this map, the field in
+  // the request that matches the value will have its value set to the value
```

Comment:
that matches the value => the gRPC request field that is specified by the map value will ...